docs/source/tutorials/create_and_submit_dataset.rst

```diff
@@ -270,6 +276,12 @@ Records file snippet::
    column. If the dataset is missing data for specific dimension combinations, include a row for
    each combination and set ``id`` to ``null``.
 
+.. warning:: All dimension columns must be strings, including columns that look like numbers, such
```

Comment:
GitHub may have hidden some required context. Does the full content change your opinion?
```
5. Create ``load_data_lookup.parquet``. The ``id`` column should match the values in
   ``load_data.parquet`` so that a single table can be produced by joining the two tables on that
   column. If the dataset is missing data for specific dimension combinations, include a row for
   each combination and set ``id`` to ``null``.

.. warning:: All dimension columns must be strings, including columns that look like numbers, such
   as ``model_year``.

.. warning:: If your dataset uses FIPS county codes, be sure to not inadvertently drop leading
   zeros.
```

Reply:
Not exactly, the initial issue I had was that it took me a moment to remember that there are two or three types of columns: dimension columns, value columns, columns that are neither (id, scale factor). We don't talk about value columns much, I just thought if we do this, it would be more clear:
```
warning:: All dimension columns except time, including columns that look like numbers, such as ``model_year``, are considered attributes and therefore they must be strings. Conversely, only value columns are numeric as they measure a quantity.
```